src/tasks/__init__.py

```diff
@@ -19,6 +19,18 @@
 ALL_GLUE_TASKS = ['sst', 'cola', 'mrpc', 'qqp', 'sts-b',
                   'mnli', 'qnli', 'rte', 'wnli', 'mnli-diagnostic']
 
+# Tasks for the spring19_seminar; similar to cola but write predictions differently
```

Comment:
Big general question: Will these same tasks work for the non-probing experiments? For those, we'll need to be able to train some models on CoLA then evaluate them on the test sets for these tasks, right?

If we definitely need two different sets of NPI tasks, make it clearer in comments which ones are for which experiments. You may need to check in with @PterosDiacos.

Reply:
I think it depends on what our BERT+cola model (and BERT_ccg+cola, BERT_mnli+cola...etc) is doing. Is cola a pretraining step or a target-task training step?